src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,9 +1,19 @@
 package core.mate.academy.model;
 
-public abstract class Machine implements Workable {
+import core.mate.academy.service.MachineProducer;
+
+public abstract class Machine implements Workable, MachineProducer {
```

Comment:
[you shouldn't implement MachineProducer here, this interface should be implemented by](https://github.com/mate-academy/jv-pecs#:~:text=There%20is%20MachineProducer,the%20suitable%20option.)